### ISSUE TITLE
rfc: React Native and FAST

### ISSUE THREAD

**Issue by amoghkulkarnifr** [microsoft/fast#6873]
# 💬 RFC

Hello Devs and community, sorry if this is not a good place to ask this - I am trying to gather information on whether FAST can be used to create components for React Native applications. Is there a way to use FAST design system with React Native? 

## 🔦 Context

We are a team working on a web application and mobile applications (which are to be developed using React Native) and we want to create our own component library and a design system to provide a consistent UI/UX across platforms. The idea is to create and maintain a single component library for both our web application and our mobile applications. 

We were looking at FAST as a potential tool to achieve this, but we could not get sufficient information about whether FAST can be integrated with React Native development workflow. 

---
TL/DR; Any resources related to if/how FAST can be used with React Native will be helpful. Thank you in advance!

**Comment by janechu** [comment]
FAST core objectives are for the browser, so we will not be looking into platform native solutions at this time.
